Starting GS2 runs
nzgrid=32 npol=2 nstep=12000 dt=0.01 nhermite=16 nlaguerre=8 growth_rate=0.302905 took 9.656207s
[login06:69638] *** Process received signal ***
[login06:69638] Signal: Segmentation fault (11)
[login06:69638] Signal code: Address not mapped (1)
[login06:69638] Failing at address: (nil)
[login06:69638] [ 0] linux-vdso64.so.1(__kernel_sigtramp_rt64+0x0)[0x2000000504d8]
[login06:69638] [ 1] python3(PyObject_GC_UnTrack+0x24)[0x1377cb194]
[login06:69638] [ 2] python3(_Py_Dealloc+0x28)[0x1376ab738]
[login06:69638] [ 3] python3(+0xc5680)[0x1376a5680]
[login06:69638] [ 4] python3(_PyTrash_thread_destroy_chain+0x5c)[0x1376ab6cc]
[login06:69638] [ 5] python3(+0x5f710)[0x13763f710]
[login06:69638] [ 6] python3(_Py_Dealloc+0x28)[0x1376ab738]
[login06:69638] [ 7] python3(+0x1e9a94)[0x1377c9a94]
[login06:69638] [ 8] python3(+0x1ea254)[0x1377ca254]
[login06:69638] [ 9] python3(+0x1ea738)[0x1377ca738]
[login06:69638] [10] python3(_PyObject_GC_New+0x38)[0x1377cb238]
[login06:69638] [11] python3(+0xb0d40)[0x137690d40]
[login06:69638] [12] python3(PyObject_GetIter+0x34)[0x137875624]
[login06:69638] [13] python3(+0x98844)[0x137678844]
[login06:69638] [14] python3(+0x99afc)[0x137679afc]
[login06:69638] [15] python3(+0xef24c)[0x1376cf24c]
[login06:69638] [16] python3(_PyObject_MakeTpCall+0xec)[0x13766077c]
[login06:69638] [17] python3(_PyEval_EvalFrameDefault+0x834c)[0x137648cec]
[login06:69638] [18] python3(PyEval_EvalFrameEx+0x34)[0x137746564]
[login06:69638] [19] python3(_PyEval_EvalCodeWithName+0xec0)[0x1377474a0]
[login06:69638] [20] python3(_PyFunction_Vectorcall+0xcc)[0x13766032c]
[login06:69638] [21] python3(_PyEval_EvalFrameDefault+0x8590)[0x137648f30]
[login06:69638] [22] python3(PyEval_EvalFrameEx+0x34)[0x137746564]
[login06:69638] [23] python3(+0x5f1d4)[0x13763f1d4]
[login06:69638] [24] python3(_PyEval_EvalFrameDefault+0x8590)[0x137648f30]
[login06:69638] [25] python3(PyEval_EvalFrameEx+0x34)[0x137746564]
[login06:69638] [26] python3(+0x5f1d4)[0x13763f1d4]
[login06:69638] [27] python3(+0x2aa53c)[0x13788a53c]
[login06:69638] [28] python3(_PyEval_EvalFrameDefault+0x6e90)[0x137647830]
[login06:69638] [29] python3(PyEval_EvalFrameEx+0x34)[0x137746564]
[login06:69638] *** End of error message ***
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun noticed that process rank 0 with PID 69638 on node login06 exited on signal 11 (Segmentation fault).
--------------------------------------------------------------------------
